src/Scheduler.php

```diff
@@ -215,6 +215,12 @@ public function schedule(Task $task, Frequency $frequency): self
                 $this->emit(static::ON_TASK_RUN, [$task, $promise]);
             });
             $this->emit(static::ON_TASK_SCHEDULED, [$task, $now]);
+
+            if ($frequency instanceof OneOff) {
+                // One-time tasks are, as the name implies, a one-time job, so no need
```

Comment:
Please remove comment. It's enough to have a commit description that explains the change.